### ISSUE TITLE
System.IO.FileNotFoundException at TuaRua.FreSharp.FreSharpHelper.DispatchEvent

### ISSUE THREAD

**Issue by TommyTheBlackbird** [tuarua/FreSharp#10]
This error is driving my crazy... In my dev computer everything works perfect, but when I run my app in the client test computer (*Windows 8.1 Pro 32-bit*), I get this exception:

```
System.IO.FileNotFoundException: Could not load file or assembly 'FreSharpCore.dll' or one of its dependencies. The specified module could not be found.
File name: 'FreSharpCore.dll'
   at TuaRua.FreSharp.FreSharpHelper.DispatchEvent(IntPtr& freContext, String name, String value)
   at PlayerBridgeANELib.MainController.RaiseEvent(String eventType, String jsonEventInfo)
   at PlayerBridgeANELib.MainController.NotifyException(Exception ex)
   at PlayerBridgeANELib.MainController.WatchSpontania()
   at PlayerBridgeANELib.MainController.InitController(IntPtr ctx, UInt32 argc, IntPtr[] argv)
``` 
The exception line is in my code is in the ```SendEvent(...)``` method call.

* .Net 4.6 installed ✔️ 
* VC++ redist installed ✔️ 
* My C# ANE lib dll, ```FreeSharp.dll```, ```FreSharpCore.dll``` in the .exe folder ✔️ 

If the DispatchEvent method is reached, I guess ```FreSharpCore.dll``` is loaded but not one of it dependencies, but ```FreSharp.dll``` is in the same folder. So which is the missing file? Or maybe I have to place dlls in other location?

The Windows Event viewer and the Windows Error Reporting file don't give any useful information (or I don't interpreting it properly...).

Any clue is welcome.

**Comment by tuarua** [comment]
Well the message says it can't load FreSharpCore.dll, you say you guess it is already loaded but are you making any other FRE calls before the sendEvent ? - if not that it is not certain it is loaded.

Are you building your C++ dll with v140 targeting 8.1 like the sample projects?
FreSharpCore.dll uses VC++ Dist 2015
That and .NET is Core's only dependency. 

You are sending an event when an Exception occurs. What is this exception? are you able to run a very basic ANE on the machine.

I have had no problems using FreSharp on other non-dev machines.

**Comment by TommyTheBlackbird** [comment]
You nailed it. After struggling a lot I realized that I was using v120 (cause I use VS 2013) and inspecting the FreSharp source project I saw it uses v140. 

I installed VC++ redist 2015 and worked again. I only wish the error messages where more clear...

Anyway, it seems that it is not necessary to compile the ANE lib with v140 to work, but having VC++ 2015 is mandatory for ```FreSharp.dll``` to run. At least for now... may later I can find new problems, but right now I have no chance to upgrade to VS 2015 and is not possible to install and compile with v140 with VS 2013.

Thanks again for your help.
